db-scripts/src/main/resources/cgds.sql

```diff
@@ -775,6 +791,22 @@ CREATE TABLE `clinical_event_data` (
 );
 
 -- --------------------------------------------------------
+DROP TABLE IF EXISTS `reference_genome_gene`;
+CREATE TABLE `reference_genome_gene` (
+    `ENTREZ_GENE_ID` int(11) NOT NULL,
+    `CYTOBAND` varchar(64) NOT NULL,
+    `EXONIC_LENGTH` int(11) DEFAULT NULL,
+    `GENE_START` bigint(20) DEFAULT NULL,
```

Comment:
Should we normalize these lengths with copy_number_seg, START/END?

Reply:
I am not quite clear how is currently calculated. Pieter or JJ might be better person to answer this. JJ proposed this field.

Reply:
sorry, now I got what you mean. It might be better we normalize them with copy_number_seg. changed!

Reply:
Yes, you have columns in reference_genome_gene that represent (I think) similar columns in both gene and copy_number_seg tables, but with different names, types, and lengths.  I think the fields in reference_genome_gene should get updated to have the same names, types, and lengths already agreed upon in the existing tables?  For example, I still see START, END as bigint(20) in reference_genome_gene, but they are defined in copy_number_seg as int(11).

Reply:
Actually, this was an incorrect comment - I was equating a copy number segment length with the reference genome gene length.  You can disregard this.